packages/react-composites/src/composites/MeetingComposite/adapter/MeetingAdapter.ts

```diff
@@ -14,15 +14,15 @@ import {
   ParticipantLeftListener
 } from '../../CallComposite';
 import { ChatAdapter, MessageReadListener, MessageReceivedListener, MessageSentListener } from '../../ChatComposite';
-import { MeetingState } from '../state/MeetingState';
+import { MeetingClientState } from '../state/MeetingState';
 
 export type ConflictingProps = 'getState' | 'onStateChange' | 'offStateChange' | 'on' | 'off';
 
 export interface MeetingAdapter extends Omit<ChatAdapter, ConflictingProps>, Omit<CallAdapter, ConflictingProps> {
   // Meeting-specific interfaces
-  getState(): MeetingState;
-  onStateChange(handler: (state: MeetingState) => void): void;
-  offStateChange(handler: (state: MeetingState) => void): void;
+  getState(): MeetingClientState;
```

Comment:
What was the reasoning behind calling this `MeetingClientState`.

I noticed that we're inconsistent in this naming already. `ChatAdapter` calls this `ChatState` while `CallAdapter` calls it `CallAdapterState`. But in both cases, the `*ClientState` refers to the stateful client state.

I'd vote to call this `MeetingAdapterState` (and also change `ChatState` to `ChatAdapterState`.

Reply:
Yep, I agree, I'll update this to MeetingAdapterState (which has a high probability of becoming `MeetingAdapterState = MeetingAdapterUiState & MeetingAdapterClientState` same as chat and calling)

Reply:
Updated to `MeetingAdapterState` (I'll try remember to update chat in another PR sometime...)